#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_readme.dpatch by Petter Reinholdtsen <pere@hungry.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add URL to upstream source in README.

@DPATCH@
diff -ur insserv-1.04.0/README insserv-1.04.0/README
--- insserv-1.04.0/README	2005-09-14 17:53:05.000000000 +0200
+++ insserv-1.04.0/README	2005-11-10 21:47:38.000000000 +0100
@@ -5,6 +5,12 @@
 boot scripts used by the system during boot time),
 type `make' and `make install'.
 
+   Downloading latest version
+   ==========================
+
+The latest version of the insserv source is available from
+<URL: ftp://ftp.suse.com/pub/projects/init/ >.
+
    Processes controlling
    =====================
 
